TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -166,6 +166,11 @@ public static void ScanTokensLine(TokensLine tokensLine, MultilineScanState init
             Token nextToken = null;
             while((nextToken = scanner.GetNextToken()) != null)
             {
+                if (nextToken.TokenType == TokenType.AlphanumericLiteral && (!nextToken.HasOpeningDelimiter || !nextToken.HasClosingDelimiter))
+                {
+                    tokensLine.AddDiagnostic(MessageCode.SyntaxErrorInParser,
+                        tokensLine.Indicator.StartIndex, tokensLine.Indicator.EndIndex, "Literal is not well delimited.");
```

Comment:
"is not correctly delimited" ?

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1604/commits/5de0690f865a31192983ee6452e96a892455d316